bitnami/victoriametrics/templates/vmagent/dep-ds.yaml

```diff
@@ -0,0 +1,209 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.vmagent.enabled }}
+{{- $kind := lower .Values.vmagent.kind }}
+{{- if eq $kind "daemonset" }}
+apiVersion: {{ include "common.capabilities.daemonset.apiVersion" . }}
+kind: DaemonSet
+{{- else }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+{{- end }}
+metadata:
+  name: {{ template "victoriametrics.vmagent.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  {{- $versionLabel := dict "app.kubernetes.io/version" ( include "common.images.version" ( dict "imageRoot" .Values.vmagent.image "chart" .Chart ) ) }}
```

Comment:
Is this required? AFAIK the versioning is synced on VictoriaMetrics components. Please check this on every manifest.

Reply:
Fixed